io.github.brunoherbelin.Vimix.json

```diff
@@ -0,0 +1,159 @@
+{
+    "app-id": "io.github.brunoherbelin.Vimix",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "43",
+    "sdk": "org.gnome.Sdk",
+    "command": "vimix",
+    "rename-desktop-file": "vimix.desktop",
+    "rename-icon": "vimix",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=ipc",
+        "--share=network",
+        "--device=dri",
+        "--device=shm",
```

Comment:
This doesn't do what you think it does.

Why do you think you need it?

Reply:
I am ok to reduce but
- `network` is needed for in & out streaming
- device `dri` is recommended for opengl driver, no?
- device shm is needed for shared-memory with gstreamer (shmsink, shmsrc, shmdatasink and shmdatasrc): i tested without and it was not working